packages/server/modules/workspaces/domain/operations.ts

```diff
@@ -29,15 +29,27 @@ export type GetWorkspaces = (args: {
 
 /** Workspace Roles */
 
-export type GetWorkspaceCollaborators = (args: {
+type GetWorkspaceCollaboratorsArgs = {
   workspaceId: string
-  /**
-   * Optionally filter by role
-   */
-  role?: WorkspaceRoles
-}) => Promise<
-  Array<UserWithRole<LimitedUserRecord> & { workspaceRole: WorkspaceRoles }>
->
+}
+
+type GetWorkspaceCollaboratorsOptions = {
+  filter?: {
+    /**
+     * Optionally filter by workspace role
+     */
+    role?: string
+    /**
+     * Optionally filter by user name or email
+     */
+    search?: string
+  }
+}
+
+export type GetWorkspaceCollaborators = (
+  args: GetWorkspaceCollaboratorsArgs,
+  opts: GetWorkspaceCollaboratorsOptions
```

Comment:
we need to make this optional and partial

Reply:
why are you always so mean to me